plaso/parsers/czip_plugins/oxml.py

```diff
@@ -0,0 +1,320 @@
+      if not property_name:
+        property_name = self._FormatPropertyName(name)
+
+      properties[property_name] = xml_element.text
+
+    return properties
+
+  def _ParseRelationshipsXMLFile(self, xml_data):
+    """Parses the relationships XML file (_rels/.rels).
+
+    Args:
+      xml_data (bytes): data of a _rels/.rels XML file.
+
+    Returns:
+      list[str]: property file paths. The path is relative to the root of
+          the ZIP file.
+
+    Raises:
+      zipfile.BadZipfile: if the relationship XML file cannot be read.
+    """
+    xml_root = ElementTree.fromstring(xml_data)
+
+    property_files = []
+    for xml_element in xml_root.iter():
+      type_attribute = xml_element.get('Type')
+      if 'properties' in repr(type_attribute):
+        target_attribute = xml_element.get('Target')
+        property_files.append(target_attribute)
+
+    return property_files
+
+  def _ProduceEvent(
+      self, parser_mediator, event_data, properties, property_name,
+      timestamp_description, error_description):
+    """Produces an event.
+
+    Args:
+      parser_mediator (ParserMediator): mediates interactions between parsers
+          and other components, such as storage and dfvfs.
+      event_data (OpenXMLEventData): event data.
+      properties (dict[str, object]): properties.
+      property_name (str): name of the date and time property.
+      timestamp_description (str): description of the meaning of the timestamp
+          value.
+      error_description (str): description of the meaning of the timestamp
+          value for error reporting purposes.
+    """
+    time_string = properties.get(property_name, None)
+    if not time_string:
+      return
+
+    # Date and time strings are in ISO 8601 format either with 1 second
+    # or 100th nano second precision. For example:
+    # 2012-11-07T23:29:00Z
+    # 2012-03-05T20:40:00.0000000Z
+    date_time = dfdatetime_time_elements.TimeElements()
+
+    try:
+      date_time.CopyFromStringISO8601(time_string)
+
+      event = time_events.DateTimeValuesEvent(date_time, timestamp_description)
+      parser_mediator.ProduceEventWithEventData(event, event_data)
+    except ValueError as exception:
+      parser_mediator.ProduceExtractionError(
+          'unsupported {0:s}: {1:s} with error: {2!s}'.format(
+              error_description, time_string, exception))
+
+  def Process(self, parser_mediator, archive_proxy, **kwargs):
+    """Parses an OXML file-like object.
+
+    Args:
+      parser_mediator (ParserMediator): mediates interactions between parsers
+          and other components, such as storage and dfvfs.
+      archive_proxy (parsers.CompoundZIPParser): a slim proxy giving simple
+          access to archive files.
+
+    Raises:
+      UnableToParseFile: when the file cannot be parsed.
+    """
+
+    # First of all, call upper processing logic
+    super(OpenXMLPlugin, self).Process(parser_mediator, archive_proxy, **kwargs)
```

Comment:
Rather than overloading Process here, just create a different method that's called from the Process() implementation in the interface - this is the way the other parsers-that-have-plugins work.